birdhouse/components/jupyterhub/default.env

```diff
@@ -72,26 +72,6 @@ export JUPYTERHUB_AUTHENTICATOR_REFRESH_AGE=60
 # Usernames that should be given admin access in jupyterhub
 export JUPYTERHUB_ADMIN_USERS='{\"${MAGPIE_ADMIN_USERNAME}\"}'  # python set syntax
 
-# Resource limits for JupyterLab containers. Resource limits can be set per Magpie user or group.
-# The value for this variable is a whitespace delimited string. Each section is delimited by colons (:) 
-# where the first element is either `group` or `user` and the second element is the name of the user or group
-# to apply the limits to. The rest are resource limits of the form `limit=amount`. For example:
-#
-#  export JUPYTERHUB_RESOURCE_LIMITS="
-#     user:user1:mem_limit=30G
-#     group:group1:mem_limit=10G:cpu_limit=1
-#     group:group2:cpu_limit=3
-#  "
-#
-#  Supported limits are: `mem_limit` and `cpu_limit`. See the Jupyterhub Dockerspawner documentation
-#  for details and supported values.
-#  Note that this will not create the groups in Magpie, that must be done manually.
-#  Note that if a user belongs to multiple groups, later values in `JUPYTERHUB_RESOURCE_LIMITS` will take
-#  precedence. For example, if a user named user1 belongs to group1 and group2 then the following limits will apply:
-#  - mem_limit=10G (because group1 is later in the list)
-#  - cpu_limit=3 (because group2 is later in the list)
-export JUPYTERHUB_RESOURCE_LIMITS=
```

Comment:
@mishaschwartz 

Thinking about this last night, can you just restore the default empty value here?

All default values should be in a corresponding `default.env`, like all other vars, to not confuse users and to maintain consistency.  We should not expect the user to search the code for that default value.

The commented out value in `env.local.example` is just an example, it should not be considered a default value.

If you want to avoid duplicating all the documentations for that var in `default.env`, you can simply reference the user to the `env.local.example`.

Reply:
> Sure, we really need to write down some of these policies somewhere

Absolutely agree, added https://github.com/bird-house/birdhouse-deploy/pull/620 before I forget.